logbook/logbook.go

```diff
@@ -211,6 +212,57 @@ func (book *Book) DeleteAuthor() error {
 	return fmt.Errorf("not finished")
 }
 
+// ResolveRef finds the identifier & head path for a dataset reference
+// implements resolve.NameResolver interface
+func (book *Book) ResolveRef(ctx context.Context, ref *dsref.Ref) error {
```

Comment:
I don't think logbook should be relied on at all for resolving references. It conceivably can, by iterating over each branch and collapsing them, but it's not designed for that, and I can anticipate many problems arising from using this method. If this code is never meant to be called, it's effectively dead code, and its existence can lead to mistakes, confusion, and increased maintenance cost.

As mentioned in https://github.com/qri-io/qri/issues/1305, logbook should be accepting initIDs for its methods, and therefore needs some method in the interim that converts refs to initIDs, but that should be treated as temporary, and not an intrinsic part of how it is to be used.

Let's say, in the near future, we implement branching and rewinding to old versions. How well does logbook handle that?